src/com/projectkorra/projectkorra/command/HelpCommand.java

```diff
@@ -140,21 +141,28 @@ else if (args.size() == 0) {
 			if (!ability.getInstructions().isEmpty()) {
 				sender.sendMessage(ChatColor.GOLD + usage + ability.getInstructions());
 			}
+			if (ability instanceof AddonAbility) {
+				AddonAbility abil = (AddonAbility) CoreAbility.getAbility(arg);
+				
+				sender.sendMessage(color + "* ADDON ABILITY *");
```

Comment:
This should be a configurable option in the language file. We will need to talk about this in more detail in #devs.